src/plugins/gr-he-common/core/shell.py

```diff
@@ -103,5 +103,35 @@ def _setup(self):
                 if not self.environment[ohostedcons.CoreEnv.TMUX_PROCEED]:
                     raise context.Abort('Aborted by user')
 
+        if (
+            not self.environment[ohostedcons.NetworkEnv.FORCE_IPV4]
+            and not self.environment[ohostedcons.NetworkEnv.FORCE_IPV6]
+        ):
+            self.logger.warning(
+                _('It has been detected that either --4 or --6 '
+                  'options were not used.\n '
+                  'On a dual stack environment, IPv6 will be selected '
+                  'and due to a limitation, hosted-engine will be deployed '
+                  'as IPv6 only.\n '
+                  'Make sure that DNS returns only IPv6 '
+                  'or change Java and imageio address resolution.')
```

Comment:
Not sure the text is very clear. Perhaps consult a doc writer, or shorten and write something longer in a web page. Last sentence should probably be conditional ("If you want to use IPv6, ...").